liferay-tango/liferay-portal
# LPS 145980 Move React components to compotents folder

Motivation
=======
We want to improve the organization of the javascript code
[Link to story or ticket](https://issues.liferay.com/browse/LPS-145980)

Proposed Solution
========
We have moved all the React components to the components folder. Also grouped all components used only in AuditGraphApp to the AuditGraphApp folder.

Steps to Verify:
----------------
1. Go to the Content Dashboard and check that everything works fine

Change summary (optional):
---------------
* LPS-145980 Move content dashboard components to the components folder
* LPS-145980 Move AuditGraphApp component to AuditGraphApp subfolder
